CONTRIBUTING.md

```diff
@@ -19,8 +19,8 @@ You need to install the following dependencies:
   * Linux or macOS. Windows support is coming soon. <!-- TODO: Link Java roadmap when it is published -->
   * [JDK 1.8+](https://jdk.java.net/12/).
   * [Maven 3.5+](https://maven.apache.org/download.cgi).
-  * [Rust 1.32.0](https://www.rust-lang.org/).
-  To install a specific Rust version, use `rustup install 1.32.0` command.
+  * [Stable Rust](https://www.rust-lang.org/).
+  To install a stable Rust version, use `rustup install stable` command.
```

Comment:
I think this line assuming the reader has rustup may be removed — if they have rustup, they probably have `stable`.

Reply:
Not always the case - some of the users use nightly, so I'd keep it here

Reply:
Don't hackers know how to switch versions? :upside_down_face: